public/luna_logo.js

```diff
@@ -0,0 +1,297 @@
+class Text {
```

Comment:
This file should be loaded here as library. If we change how we draw the logo, we will not copy-paste the code here. You have to use luna-logo as npm library instead.